test/acceptance/tests/connect/connect_inject_namespaces_test.go

```diff
@@ -198,7 +198,7 @@ func TestConnectInjectNamespaces(t *testing.T) {
 				}
 
 				logger.Log(t, "creating intention")
-				_, _, err := consulClient.Connect().IntentionCreate(intention, nil)
+				_, err := consulClient.Connect().IntentionUpsert(intention, nil)
```

Comment:
I'm just curious, why did this end up changing?

Reply:
`IntentionCreate` has been deprecated by the api, so now it's `IntentionUpsert`!
Kinda cool that the linter checks for this and flags it! :)